tsconfig.json

```diff
@@ -0,0 +1,84 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "incremental": true,                   /* Enable incremental compilation */
+    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
+    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
+    "lib": [
+      "es2020",
+      "dom"
+    ],
+    "allowJs": true,                       /* Allow javascript files to be compiled. */
+    "checkJs": true,                       /* Report errors in .js files. */
+    "jsx": "react",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
+    "declaration": true,                   /* Generates corresponding '.d.ts' file. */
+    // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
```

Comment:
Are these commented out because they are default values?

Reply:
yeah, when you run `tsc --init` it writes a config file like this with a whole bunch of values. I suppose we could delete the commented-out ones - that might be cleaner

Reply:
No, it's fine to leave it. I think most config files have a similar convention.